lib/unitarize_force_quda.cu

```diff
@@ -1,6 +1,10 @@
 #include <cstdlib>
 #include <cstdio>
-
+#include <iostream>
```

Comment:
Why all these includes beyond just the backend?

Reply:
Fixed, two of them were repeated.